integration-tests/api-gateway/contract/utils/PactJwtFormatter.ts

```diff
@@ -0,0 +1,42 @@
+import { base64url, decodeJwt, decodeProtectedHeader } from "jose";
+
+export const formatJwtForPactTest = (body: string) => {
+  const decodedVc = decodeJwt(body);
+  const stringifyVc = JSON.stringify(decodedVc);
+  const parseVc = JSON.parse(stringifyVc);
+
+  const vcWithReplacedFields = replaceDynamicVcFieldsIfPresent(parseVc);
+  const reorderedVc = reorderVc(vcWithReplacedFields);
+
+  return (
+    getJwtHeader(body) +
+    "." +
+    base64url.encode(JSON.stringify(reorderedVc)) +
+    "."
+  );
+};
+
+const replaceDynamicVcFieldsIfPresent = (parseVc: any) => {
+  parseVc.nbf = parseVc.nbf == null ? parseVc.nbf : 4070908800;
+  parseVc.iss = parseVc.iss == null ? parseVc.iss : "dummyNinoComponentId";
+  parseVc.exp = parseVc.exp == null ? parseVc.exp : 4070909400;
+  parseVc.jti = parseVc.jti == null ? parseVc.jti : "dummyJti";
+  return parseVc;
+};
+
+const reorderVc = (vc: any) => ({
```

Comment:
Interesting that we need this but BAV doesn't! Thought it might be ts related but I see that removing this fails so must be something else